### ISSUE TITLE
Issue on theoretic result corresponding to Zhang Chiyuan ICLR-17 paper

### ISSUE THREAD

**Issue by guixianjin** [chenpf1025/noisy_label_understanding_utilizing#2]
Hi! Thank you for your interesting paper. But I have a small issue.
In this paper, it says "Interestingly, Eq.(4) perfectly fits the experimental results of generalization accuracy shown in Fig. 1(c) of (Zhang et al. 2017), ..."

But in  (Zhang et al. 2017), it seems that the test accuracy is tested on clean labels. 
And in this paper, Eq.(4): P(y^f = y) = (1-epsilon)^2 + epsilon^2/(c-1),  y is noisy label according to notation in this paper. 
So maybe they can't correspond to each other.

**Comment by chenpf1025** [comment]
Hi! Thank you for your interest.

In (Zhang et al. 2017), they also study the generalization performance in the presence of label noise and the test set should have the same noise as the training set, which implies that a model trained with noisy labels with 100% training accuracy can not generalize even if the test set has the same noise.

When test on a clean test set, we found that the test accuracy should be linear to the noise ratio.

**Comment by guixianjin** [comment]
OKay, thank you.
